src/api/file_common_dataset.yaml

```diff
@@ -98,3 +98,15 @@ info:
         type: string
         description: The organism of the sample in the dataset.
         required: false
+      - name: parameter_flowsom_xdim
+        type: integer
+        description: Parameter used to define the dimensions of the self-organizing map (SOM) grid used for flowsom clustering.
+        required: true
+      - name: parameter_flowsom_ydim
+        type: integer
+        description: Parameter used to define the dimensions of the self-organizing map (SOM) grid used for flowsom clustering.
+        required: true
+      - name: parameter_flowsom_nclus
+        type: integer
+        description: Parameter used to define the number of clusters used in the meta-clustering step of the flowsom algorithm.
+        required: true 
```

Comment:
Anybody who wants to add a dataset to this benchmark would need to provide a reasonable value for these metrics. Could you extend the description a little bit to give some instructions / a guide on how to define these values?